redirects.yaml

```
@@ -37,370 +37,370 @@ tutorials/how-to-work-with-bundles-in-charmcraft/?: /docs/sdk/manage-bundles
+docs/juju/telemetry-and-juju/?: https://documentation.ubuntu.com/juju/latest/user/reference/telemetry/
 docs/juju/terraform-juju-client/?: https://canonical-terraform-provider-juju.readthedocs-hosted.com/en/latest/tutorial/
-docs/juju/the-juju-dashboard/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/juju-dashboard/
-docs/juju/the-juju-web-cli/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/juju-web-cli/
-docs/juju/troubleshoot-your-deployment/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-your-deployment/troubleshoot-your-deployment/#
-docs/juju/tutorial/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/tutorial/
-docs/juju/unit/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/unit/
-docs/juju/upgrade-your-juju-deployment/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-your-deployment/upgrade-your-deployment/
-docs/juju/upgrading/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/upgrading-things/
-docs/juju/user/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/user/
-docs/juju/user-permissions/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/user/#user-access-levels
-docs/juju/vmware-vsphere/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/cloud/list-of-supported-clouds/the-vmware-vsphere-cloud-and-juju/
-docs/juju/worker/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/worker/
-docs/sdk/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/howto/manage-charms/#build-a-charm
-docs/sdk/12-factor-app-charm/?: https://canonical-juju.readthedocs-hosted.com/en/latest/user/reference/charm/charm-taxonomy/#factor-app-charm
+docs/juju/the-juju-dashboard/?: https://documentation.ubuntu.com/juju/latest/user/reference/juju-dashboard/
+docs/juju/the-juju-web-cli/?: https://documentation.ubuntu.com/juju/latest/user/reference/juju-web-cli/
+docs/juju/troubleshoot-your-deployment/?: https://documentation.ubuntu.com/juju/latest/user/howto/manage-your-deployment/troubleshoot-your-deployment/#
+docs/juju/tutorial/?: https://documentation.ubuntu.com/juju/latest/user/tutorial/
+docs/juju/unit/?: https://documentation.ubuntu.com/juju/latest/user/reference/unit/
+docs/juju/upgrade-your-juju-deployment/?: https://documentation.ubuntu.com/juju/latest/user/howto/manage-your-deployment/upgrade-your-deployment/
+docs/juju/upgrading/?: https://documentation.ubuntu.com/juju/latest/user/reference/upgrading-things/
+docs/juju/user/?: https://documentation.ubuntu.com/juju/latest/user/reference/user/
+docs/juju/user-permissions/?: https://documentation.ubuntu.com/juju/latest/user/reference/user/#user-access-levels
+docs/juju/vmware-vsphere/?: https://documentation.ubuntu.com/juju/latest/user/reference/cloud/list-of-supported-clouds/the-vmware-vsphere-cloud-and-juju/
```

This link is giving a 404 - it seems to have changed
```suggestion
docs/juju/vmware-vsphere/?: https://documentation.ubuntu.com/juju/latest/reference/cloud/list-of-supported-clouds/the-vmware-vsphere-cloud-and-juju/#cloud-vsphere
```